pwiz_tools/Skyline/Model/DocSettings/SrmSettings.cs

```diff
@@ -2051,6 +2051,12 @@ public SrmSettings RemoveUnsupportedFeatures(DocumentFormat documentFormat)
                             Array.Empty<string>())));
             }
 
+            if (documentFormat < DocumentFormat.VERSION_22_13)

```

Comment:
I'd rather see something more descriptive, in the style of DocumentFormat.FLAT_CROSSLINKS instead of a hardcoded version number